docs/src/development/windows.md

```diff
@@ -86,3 +86,11 @@ Before reporting the issue, make sure that you have the latest rustc version wit
 ### Cargo errors claiming that a dependency is using unstable features
 
 Try `cargo clean` and `cargo build`.
+
+### `STATUS_ACCESS_VIOLATION`
+
+This error can happen if you are using the "rust-lld.exe" linker. Consider trying a different linker.
+
+If you are using a global config. Consider moving zed to a nested directory and add a `.cargo/config.toml` with a custom linker config in the parent directory.
+
+See this issue for more information https://github.com/zed-industries/zed/issues/12041
```

Comment:
Can we make this a proper Markdown link?

Reply:
Do you prefer `<url>` or `[url](url)`?

Reply:
> Do you prefer `<url>` or `[url](url)`?

`[url](url)`

Reply:
done